docs/getting-started.rst

```diff
@@ -174,117 +158,71 @@ Note that if one of these columns exists, it will be overwritten.
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld_hh      |                                                     | Float      |
+|gross_e6_tu        |Taxable Inc. from Rents, couple sum                  | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|kindergeld_tu      |                                                     | Float      |
+|gross_e7_tu        |Taxable Inc. from Pensions, couple sum               | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|uhv                |                                                     | Float      |
+|abgst_tu           |Monthly Capital Income Tax Due, couple sum           | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|wohngeld_basis     |                                                     | Float      |
+|abgst              |Monthly Capital Income Tax Due, individual           | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|wohngeld_basis_hh  |                                                     | Float      |
+|soli               |Monthly Solidarity Surcharge due, individual         | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|ar_alg2_ek_hh      |                                                     | Float      |
+|soli_tu            |Monthly Solidarity Surcharge due, couple sum         | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|alg2_grossek_hh    |                                                     | Float      |
+|kindergeld         |Monthly Child Benefit, individual to the child       | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|mehrbed            |                                                     | Float      |
+|kindergeld_tu      |Monthly Child Benefit, household sum                 | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|regelbedarf        |                                                     | Float      |
+|incometax          |Monthly Income Tax Due, individual                   | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|regelsatz          |                                                     | Float      |
+|incometax_tu       |Monthly Income Tax Due, couple sum                   | Float      |
 +-------------------+-----------------------------------------------------+------------+
-|alg2_kdu           |                                                     | Float      |
+|uhv                |Alimony Advance payment, individual to the child     | Float      |
```

Comment:
What is 'individual to the child' supposed to mean? This is the government paying because somebody else does not, right?

Reply:
Changed to "assigned to the child"

Reply:
the "individual" refers to the recipient and to the unit it should be assigned in the data set (as opposed to a household sum).

Reply:
Does "assigned to the child" work for everybody?

Reply:
Just leave as 'individual' for now.

Btw, please add a note to the docs (this PR is as good as any) before this table on the data structure. I.e., they need to be in "fully" long format: Each household member gets his/her own row, regardless of age.

Reply:
Inserted the note and changed the description to individual